degirum_tools/notifier.py

```diff
@@ -142,9 +165,18 @@ def analyze(self, result):
             ):
                 result.notifications[self._name] = self._message.format(result=result)
 
-                if self._token is not None:
+                if self._token is not None and self._apprise_obj is not None:
                     # TODO: send notification to cloud
-                    pass
+
+                    
+                    # For now, assuming a hardcoded message
+                    for title, body in result.notifications.items():
+                        self._apprise_obj.notify(

```

Comment:
Since we already know, that this call is blocking and can be long, can we call it in separate process?

You can look here:
https://docs.python.org/3/library/multiprocessing.html#:~:text=channel%20between%20processes%3A-,Queues,-The%20Queue%20class

Reply:
This is the only thing which is left not fixed. I fixed all other issues

Reply:
Got it. But for future reference, how should I go about doing things like print to debug?

Reply:
You print during your debug, then you delete all prints before commit

Reply:
We can consider implementing standard logging in this library eventually